vocabularies/Common.xml

```diff
@@ -188,12 +189,12 @@
         <Annotation Term="Core.Description" String="Annotated property or parameter is a unit of measure" />
       </Term>
 
-      <Term Name="UnitSpecificScale" Type="Edm.PrimitiveType" AppliesTo="Property">
+      <Term Name="UnitSpecificScale" Type="Edm.PrimitiveType" Nullable="false" AppliesTo="Property">
```

Comment:
```suggestion
      <Term Name="UnitSpecificScale" Type="Edm.PrimitiveType" AppliesTo="Property">
```
Shall remain nullable, since it may be dynamic via `Edm.Path`. Same for the next three.

Reply:
Then we should describe how `null` is to be treated: as zero?

I'd rather require that the path points to a non-nullable property.

Reply:
If you think that null makes no sense, then yes.

Reply:
Yes, makes no sense.